lib/decompress.ps1

```diff
@@ -35,6 +54,53 @@ function Test-LessmsiRequirement {
     }
 }
 
+function Expand-ZstdArchive {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory = $true, Position = 0, ValueFromPipeline = $true)]
+        [String]
+        $Path,
+        [Parameter(Position = 1)]
+        [String]
+        $DestinationPath = (Split-Path $Path),
+        [String]
+        $ExtractDir,
+        [Parameter(ValueFromRemainingArguments = $true)]
+        [String]
+        $Switches,
+        [ValidateSet("All", "Skip", "Rename")]
+        [String]
+        $Overwrite,
+        [Switch]
+        $Removal
+    )
+
+    $TarPath = "$scoopdir\apps\git\current\usr\bin\tar.exe"
```

Comment:
This assumes that git was installed locally (`$scoopdir`) and not globally (`$globaldir`). See https://github.com/lukesampson/scoop/wiki/Pre--and-Post-install-scripts

Also, not all users will install git via scoop. I don't per https://github.com/lukesampson/scoop/issues/3877.

See also https://github.com/ScoopInstaller/Main/pull/680

Reply:
I see your points, thanks Rasa!

I pushed a change, to use zstd and the 'Expand-7zipArchive' function instead of using git-tar.